esm.mjs

```diff
@@ -8,6 +8,7 @@ export const {
   createOption,
   CommanderError,
   InvalidArgumentError,
+  InvalidArgumentError: InvalidOptionArgumentError,
```

Comment:
```suggestion
InvalidOptionArgumentError, // deprecated old name
```

Some small suggestions. I would prefer the esm just uses the cjs export, so when the export is removed from the cjs it breaks the esm. And a comment that it is deprecated.

The TypeScript uses an alias because with the stronger typing it seemed more appropriate. I don't see that as making a difference in esm?